### DISCUSSION TITLE
問い合わせがページチャット形式である必要があるか

### DISCUSSION THREAD

**Discussion by L4Ph** [aipictors/aipictors#70] [アイデア - Ideas]
このmilestoneと`(beta)/support`を見ていたのですが、そもそも問い合わせページがチャット形式である必要があるのかという部分が気になりました。

https://github.com/aipictors/aipictors/milestone/7

個人的にはチャット形式にするのであれば、Zendeskなどに一任してしまうほうがよいのではないのかなと感じています。

一般的なフォーム形式にするのであれば、幸い、shadcnには`Auto Form`があるので、これで比較的実装するべき部分を減らせるのではないかなと感じています。
https://zenn.dev/kaz_z/articles/zod-auto-form

**Comment by L4Ph** [comment]
チャット形式を維持し、画像等も使用する予定であればWYSIWYGなエディタに切り替えるのもありなのではないのかなと思っています

**Comment by nakatsuka-k** [comment]
チャット形式なのは、運営からユーザに直接連絡することもあるからですね。 
本当はDM機能の一環として作成していて
ユーザ↔ユーザの機能は、通報のシステムがまだ完成していないので公開していない感じです。

**Comment by shigurenimo** [comment]
現状の問題
- チケットの管理ができない
- 問い合わせたときに必要な情報が提供されない可能性がある(フォーム形式ほど決まった入力形式ではないため)
- 複数回同じような問い合わせが来た際に、必要な情報が分散したり、不要な情報が混じる可能性がある
